### PR TITLE
Dev > Prod

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
TW-774 Wording changes to CI integrations #4459
TW-794 cycle 8 edits #4454
TW-785 Flows copyedits #4449
TW-409 Emphasize API sync for SCM integrations #4460
TW-758 Cycle 7 edits #4451
TW-800 Azure AD SCIM attribute mapping types #4453
TW-792 Doc accessing integrations from the right sidebar #4446
TW-775 Edit note #4456
TW-744 Change order of OS in postman-cli-installation.md #4455
markeng-1950 refractor Header #4435

### CODE REVIEW

File: src/pages/docs/postman-flows/core-concepts/blocks-list.md [hunk 1]
```diff
@@ -5,124 +5,218 @@ updated: 2022-11-30
...
 ## Data blocks
 
-* **Template**
+### Template
 
-  Enables the free-hand structuring of data.
+<img alt="Template block" src="https://assets.postman.com/postman-labs-docs/all-blocks/template-block.png" width="245px"/>
 
-* **Get Variable**
+Enables the free-hand structuring of data, such as the pasting of JSON data. Also supports [FQL](../../flows-query-language/introduction-to-fql/).
```
Comment (HeatherStenson) [inline src/pages/docs/postman-flows/core-concepts/blocks-list.md:21]: I think this link might be a relic from the labs repo?

File: src/pages/docs/postman-flows/core-concepts/blocks-list.md [hunk 2]
```diff
@@ -5,124 +5,218 @@ updated: 2022-11-30
...
 
-  Retrieves the value of an assigned variable.
+**Variables**: Named data or assigned values that can be accessed using FQL.
 
-* **Create Variable**
+**Out**: Outputs the data entered in the text box.
```
Comment (HeatherStenson) [inline src/pages/docs/postman-flows/core-concepts/blocks-list.md:25]: List formatting - not important rn but can be fixed later.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 3]
```diff
@@ -85,19 +85,19 @@ You can review invite requests team members have submitted in your [team dashboa
...
+To remove a user from your team, go to your [team dashboard](https://go.postman.co/settings/team/members). Select the delete icon <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg#icon" width="12px"> next to the user's name.
 
 ![Removing a team member](https://assets.postman.com/postman-docs/v10/remove-team-member-v10.jpg)
 
-When you remove a user from your team, you will still retain access to any data they have shared with the team. Reassign their personal workspaces to another team member to ensure the team continues to have access to any unshared work done by the user being removed. Then to confirm, select **Remove Member**.
+When you remove a user from your team, you will still retain access to any data they have shared with the team. Reassign their personal workspaces to another user to ensure the team continues to have access to any unshared work done by the user being removed. Then to confirm, select **Remove Member**.
```
Comment (petewitcher) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:92]:
```suggestion
When you remove a user from your team, you will still retain access to any data they have shared with the team. Reassign their personal workspaces to another team member to ensure the team continues to have access to any unshared work done by the user being removed. Then to confirm, select **Remove Member**.
```

File: src/pages/docs/administration/managing-your-team/configuring-domain-capture.md [hunk 4]
```diff
@@ -125,14 +125,14 @@ The experience for captured users depends on their prior team status:
     * Postman support will contact your Team Admins to discuss how to approach this prior to verifying the domain you've added to your team.
     * All existing user sessions are revoked.
     * The next time the user logs in, they will be prompted to leave their existing team.
-    * They will need to contact [Postman support](https://www.postman.com/support/) to assign the Admin role to another team member and remove them from the team. The data that solely belongs to them and hasn't been shared will be moved with them.
+    * They will need to contact [Postman support](https://www.postman.com/support/) to assign the Admin role to another user and remove them from the team. The data that solely belongs to them and hasn't been shared will be moved with them.
```
Comment (petewitcher) [inline src/pages/docs/administration/managing-your-team/configuring-domain-capture.md:128]:
```suggestion
    * They will need to contact [Postman support](https://www.postman.com/support/) to assign the Admin role to another team member and remove them from the team. The data that solely belongs to them and hasn't been shared will be moved with them.
```

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 5]
```diff
@@ -107,15 +107,15 @@ Search for a specific user by typing their name or email in the search field. Yo
...
+Teams can have two support accounts at no extra cost. Support accounts are team members with only Admin or Billing roles. Team members with a Super Admin or Developer role will consume a paid seat on your team.
 
 For full details about the Postman team roles that you can assign as a Team Admin, see [Team roles](/docs/collaborating-in-postman/roles-and-permissions/#team-roles). Some roles are only available for teams with [Postman Professional or Enterprise](https://www.postman.com/pricing) plans.
 
 ### Managing partners
 
 > **[Partners are available on Postman Enterprise plans.](https://www.postman.com/pricing)**
 
-You can manage external partners similarly to internal team members. To learn more, see [Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/).
+You can manage external partners similarly to internal team users. To learn more, see [Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/).
```
Comment (petewitcher) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:118]:
```suggestion
You can manage external partners similarly to internal team members. To learn more, see [Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/).
```

File: src/pages/docs/postman-flows/core-concepts/blocks-list.md [hunk 6]
```diff
@@ -5,124 +5,218 @@ updated: 2022-11-30
...
-* **Evaluate**
+## Trigger block
 
-  Executes FQL syntax to query structured data.
+### Start
 
-* **If**
+<img alt="Start block" src="https://assets.postman.com/postman-labs-docs/all-blocks/combined-start-block.png" width="110px"/>
 
+Used as the entry point for [running flows on the cloud](../../running-flows-on-the-cloud/webhooks/). Can be configured to hold test data for testing locally by selecting the gear icon and will act as a webhook endpoint that accepts data when running on the cloud.
```
Comment (HeatherStenson) [inline src/pages/docs/postman-flows/core-concepts/blocks-list.md:115]: Another relative URL. It'll work as is but should probably be updated to absolute format at some point.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 7]
```diff
@@ -85,19 +85,19 @@ You can review invite requests team members have submitted in your [team dashboa
 
 ### Removing team members
 
-To remove a member from your team, go to your [team dashboard](https://go.postman.co/settings/team/members). Select the delete icon <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg#icon" width="12px"> next to the team member's name.
+To remove a user from your team, go to your [team dashboard](https://go.postman.co/settings/team/members). Select the delete icon <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg#icon" width="12px"> next to the user's name.
```
Comment (petewitcher) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:88]:
```suggestion
To remove a member from your team, go to your [team dashboard](https://go.postman.co/settings/team/members). Select the delete icon <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg#icon" width="12px"> next to the user's name.
```

File: src/pages/docs/administration/team-merge.md [hunk 8]
```diff
@@ -104,16 +104,16 @@ See [performing distributed migration](#performing-distributed-migration) to cho
...
 * At this point, you can [re-share and organize collections into team workspaces](/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces/#moving-elements-to-team-workspaces).
-* Once the last team member from your earlier team has migrated to your new one, the original team will be deleted along with all of the data within it. You can then begin [collaborating](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) in your new team. To learn more about collaborating in Postman, watch [Working with your team in Postman](https://youtu.be/5lscUV-Exac).
+* Once the last user from your previous team has migrated to your new one, the original team will be deleted along with all of the data within it. You can then begin [collaborating](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) in your new team. To learn more about collaborating in Postman, watch [Working with your team in Postman](https://youtu.be/5lscUV-Exac).
```
Comment (petewitcher) [inline src/pages/docs/administration/team-merge.md:109]:
```suggestion
* Once the last team member from your previous team has migrated to your new one, the original team will be deleted along with all of the data within it. You can then begin [collaborating](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) in your new team. To learn more about collaborating in Postman, watch [Working with your team in Postman](https://youtu.be/5lscUV-Exac).
```

File: src/pages/docs/administration/team-merge.md [hunk 9]
```diff
@@ -104,16 +104,16 @@ See [performing distributed migration](#performing-distributed-migration) to cho
...
-* Team members can [leave the earlier team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#leaving-a-team) and [join your new team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#joining-a-team). They can then re-import Postman data into their respective team workspaces, creating new workspaces where necessary.
+* Team members need to [export](/docs/getting-started/importing-and-exporting-data/#exporting-postman-data) their Postman data prior to moving to your new team. You can use this as an opportunity to clean up and reorganize your workspaces.
+* Users can [leave the previous team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#leaving-a-team) and [join your new team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#joining-a-team). They can then re-import Postman data into their respective team workspaces, creating new workspaces where necessary.
```
Comment (petewitcher) [inline src/pages/docs/administration/team-merge.md:114]:
```suggestion
* Team members can [leave the previous team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#leaving-a-team) and [join your new team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#joining-a-team). They can then re-import Postman data into their respective team workspaces, creating new workspaces where necessary.
```

File: src/pages/docs/administration/team-merge.md [hunk 10]
```diff
@@ -104,16 +104,16 @@ See [performing distributed migration](#performing-distributed-migration) to cho
...
     * By default, the person who imports a collection or environment is automatically assigned the Editor role (learn more about [roles](/docs/collaborating-in-postman/roles-and-permissions/)). Everyone else on the team will be assigned as a Viewer.
-* Once the last team member from your earlier team has migrated to your new one, the original team will be deleted along with all of the data within it. You can then begin [collaborating](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) in your new team. To learn more about collaborating in Postman, watch [Working with your team in Postman](https://youtu.be/5lscUV-Exac).
+* Once the last user from your previous team has migrated to your new one, the original team will be deleted along with all of the data within it. You can then begin [collaborating](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) in your new team. To learn more about collaborating in Postman, watch [Working with your team in Postman](https://youtu.be/5lscUV-Exac).
```
Comment (petewitcher) [inline src/pages/docs/administration/team-merge.md:116]:
```suggestion
* Once the last team member from your previous team has migrated to your new one, the original team will be deleted along with all of the data within it. You can then begin [collaborating](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) in your new team. To learn more about collaborating in Postman, watch [Working with your team in Postman](https://youtu.be/5lscUV-Exac).
```

File: src/pages/docs/administration/team-merge.md [hunk 11]
```diff
@@ -147,19 +147,19 @@ See [performing distributed migration](#performing-distributed-migration) to cho
...
 
-* **What happens to the earlier team once everyone leaves?**
+* **What happens to the previous team once everyone leaves?**
 
-    If it's a free team, the team is deleted after the last member leaves the team. Previous members of the deleted team will no longer have access to the team's workspaces, including personal workspaces, or any elements in them. Users can [export their Postman data](/docs/getting-started/importing-and-exporting-data/#exporting-postman-data) prior to leaving the team to ensure it stays with them.
+    If it's a free team, the team is deleted after the last user leaves the team. Previous users of the deleted team will no longer have access to the team's workspaces, including personal workspaces, or any elements in them. Users can [export their Postman data](/docs/getting-started/importing-and-exporting-data/#exporting-postman-data) prior to leaving the team to ensure it stays with them.
```
Comment (petewitcher) [inline src/pages/docs/administration/team-merge.md:156]:
```suggestion
    If it's a free team, the team is deleted after the last team member leaves the team. Previous users of the deleted team will no longer have access to the team's workspaces, including personal workspaces, or any elements in them. Users can [export their Postman data](/docs/getting-started/importing-and-exporting-data/#exporting-postman-data) prior to leaving the team to ensure it stays with them.
```

File: src/pages/docs/administration/team-merge.md [hunk 12]
```diff
@@ -147,19 +147,19 @@ See [performing distributed migration](#performing-distributed-migration) to cho
...
 
     If it's a paid team, you must contact [Postman support](https://www.postman.com/support/) to cancel the plan subscription before you can leave the team.
 
 ## Next steps
 
-Now that you know how to migrate data between teams, you might also be interested in learning how to manage team roles and set up SSO:
+After learning how to migrate data between teams, you can learn how to manage team roles and set up SSO:
 
-* To learn how to manage your team roles, invite team members to workspaces, and adjust your team size, visit [Managing your team](/docs/administration/managing-your-team/managing-your-team/).
+* To learn how to manage your team roles, invite users to workspaces, and adjust your team size, visit [Managing your team](/docs/administration/managing-your-team/managing-your-team/).
```
Comment (petewitcher) [inline src/pages/docs/administration/team-merge.md:164]:
```suggestion
* To learn how to manage your team roles, invite team members to workspaces, and adjust your team size, visit [Managing your team](/docs/administration/managing-your-team/managing-your-team/).
```

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md [hunk 13]
```diff
@@ -42,8 +42,8 @@ To access the configuration page for API definitions, do the following:
 
 Your team can turn individual security rules on or off to meet your development needs:
 
-* To turn a security rule on, select the toggle next to the rule name. You and your team members will see violations for this rule in your API's definition.
-* To turn a security rule off, select the toggle next to the rule name. You and your team members won't see violations for this rule in your API's definition.
+* To turn a security rule on, select the toggle next to the rule name. You and team members will see violations for this rule in your API's definition.
```
Comment (petewitcher) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:45]:
```suggestion
* To turn a security rule on, select the toggle next to the rule name. You and your team members will see violations for this rule in your API's definition.
```

File: src/pages/docs/running-collections/using-newman-cli/continuous-integration.md [hunk 14]
```diff
@@ -34,17 +34,17 @@ contextual_links:
 warning: false
 ---
 
-Continuous Integration (CI) is a development practice that requires developers to regularly merge code updates into a shared repository. It involves the process of automating the build and testing of code every time a developer commits code updates.
+Continuous integration (CI) is a development practice that requires developers to regularly merge code updates into a shared repository. It involves the process of automating the build and testing of code every time a developer commits code updates.
 
-You can access collections using the Postman API to run inside your Continuous Integration / Continuous Deployment (CI/CD) environments.
+You can access collections using the Postman API to run inside your Continuous integration and continuous deployment (CI/CD) environments.
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/using-newman-cli/continuous-integration.md:39]:
```suggestion
You can access collections using the Postman API to run inside your continuous integration and continuous deployment (CI/CD) environments.
```

🤦

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md [hunk 15]
```diff
@@ -42,8 +42,8 @@ To access the configuration page for API definitions, do the following:
...
+* To turn a security rule off, select the toggle next to the rule name. You and team members won't see violations for this rule in your API's definition.
```
Comment (petewitcher) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:46]:
```suggestion
* To turn a security rule off, select the toggle next to the rule name. You and your team members won't see violations for this rule in your API's definition.
```

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md [hunk 16]
```diff
@@ -81,14 +81,14 @@ Postman applies security rules configured for your [API requests](/docs/api-gove
...
+1. Select **API Security** from the team information pane.
 1. Select **Requests**.
 
 ### Turning configured rules on and off
 
 Your team can turn individual security rules on or off to meet your development needs:
 
-* To turn a security rule on, select the toggle next to the rule name. You and your team members will see violations for this rule in your API's definition.
-* To turn a security rule off, select the toggle next to the rule name. You and your team members won't see violations for this rule in your API's definition.
+* To turn a security rule on, select the toggle next to the rule name. You and team members will see violations for this rule in your API's definition.
```
Comment (petewitcher) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:91]:
```suggestion
* To turn a security rule on, select the toggle next to the rule name. You and your team members will see violations for this rule in your API's definition.
```

File: src/pages/docs/reports/team-details-reports.md [hunk 17]
```diff
@@ -68,7 +69,7 @@ The **Resource usage** report gives more information on how your team is using P
 Billed seat usage across your team.
 
 * **Billed slots** - The used billed slots compared to unused billed slots for your team.
```
Comment (HeatherStenson) [inline src/pages/docs/reports/team-details-reports.md:71]:
```suggestion
* **Billed seats** - The used billed seats compared to unused billed seats for your team.
```

File: src/pages/docs/reports/team-details-reports.md [hunk 18]
```diff
@@ -68,7 +69,7 @@ The **Resource usage** report gives more information on how your team is using P
...
-* **Unbilled slots** - The number of slots that will be billed at the next [Auto-Flex renewal](/docs/administration/billing/#utilizing-auto-flex).
+* **Unbilled slots** - The number of slots that will be billed at the next [Auto-Flex renewal](/docs/administration/billing/#using-auto-flex).
```
Comment (HeatherStenson) [inline src/pages/docs/reports/team-details-reports.md:72]:
```suggestion
* **Unbilled seats** - The number of seats that will be billed at the next [Auto-Flex renewal](/docs/administration/billing/#using-auto-flex).
```

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 19]
```diff
@@ -62,8 +62,8 @@ You can also select **Upload file(s)** to upload a new rule in valid YAML or JSO
 
 Your team can turn individual governance rules on or off to meet your development needs:
 
-* To turn a governance rule on, select the toggle next to the rule name. You and your team members will see violations for this rule in your API's definition.
-* To turn a governance rule off, select the toggle next to the rule name. You and your team members won't see violations for this rule in your API's definition.
+* To turn a governance rule on, select the toggle next to the rule name. You and team members will see violations for this rule in your API's definition.
```
Comment (petewitcher) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:65]:
```suggestion
* To turn a governance rule on, select the toggle next to the rule name. You and your team members will see violations for this rule in your API's definition.
```

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md [hunk 20]
```diff
@@ -81,14 +81,14 @@ Postman applies security rules configured for your [API requests](/docs/api-gove
...
+* To turn a security rule off, select the toggle next to the rule name. You and team members won't see violations for this rule in your API's definition.
```
Comment (petewitcher) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:92]:
```suggestion
* To turn a security rule off, select the toggle next to the rule name. You and your team members won't see violations for this rule in your API's definition.
```

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 21]
```diff
@@ -53,9 +53,9 @@ Roles define user permissions within a Postman team and a user's level of access
...
+You can [assign](/docs/administration/managing-your-team/managing-your-team/) one or more role types to users on a team, based on the functions those users require:
 
-* **Super Admin** - Manages everything within a team, including team settings, members, roles, and resources in [public, team, or private workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility). Team members with this role can perform all actions that Admin, Billing, Community Manager, API Network Manager, and Developer roles can perform ([Enterprise plans only](https://www.postman.com/pricing)).
+* **Super Admin** - Manages everything within a team, including team settings, users, roles, and resources in [public, team, or private workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility). Team members with this role can perform all actions that Admin, Billing, Community Manager, API Network Manager, and Developer roles can perform ([Enterprise plans only](https://www.postman.com/pricing)).
```
Comment (petewitcher) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:58]:
```suggestion
* **Super Admin** - Manages everything within a team, including team settings, members, roles, and resources in [public, team, or private workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility). Team members with this role can perform all actions that Admin, Billing, Community Manager, API Network Manager, and Developer roles can perform ([Enterprise plans only](https://www.postman.com/pricing)).
```

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 22]
```diff
@@ -254,6 +254,6 @@ All partners are assigned Workspace Editor or Viewer roles when invited to a Par
 
 ## Next steps
 
-Now that you understand the roles available to team members, you can manage your team's level of access and control more effectively.
+After learning about the roles available to users, you can manage your team's level of access and control more effectively.
```
Comment (petewitcher) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:257]:
```suggestion
After learning about the roles available to team members, you can manage your team's level of access and control more effectively.
```

File: src/pages/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests.md [hunk 23]
```diff
@@ -123,7 +123,7 @@ After you approve the pull request, its status in the list of pull requests is `
 
 ## Pulling updates during a pull request
 
-You can keep your forked collections and environments up to date with any changes to the parent element, for example if another team member has merged changes into the parent element.
+You can keep your forked collections and environments up to date with any changes to the parent element, for example if another user has merged changes into the parent element.
```
Comment (petewitcher) [inline src/pages/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests.md:126]:
```suggestion
You can keep your forked collections and environments up to date with any changes to the parent element, for example if another team member has merged changes into the parent element.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 24]
```diff
@@ -58,7 +58,7 @@ To convert an existing workspace to a Partner Workspace, do the following:
 
 Anyone with permissions to change workspace visibility can request to convert a workspace into a Partner Workspace, but it must be approved by a [Partner Manager](#managing-users-and-roles). If you're not a [Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) or Partner Manager, Postman will mention this and you can add an optional note to your request. Postman will notify you when a Partner Manager reviews your request and approves or denies it.
 
-> When a workspace is converted to a Partner Workspace, Postman sends in-app and email notifications to all workspace members.
+> When a workspace is converted to a Partner Workspace, Postman sends in-app and email notifications to all workspace users.
```
Comment (petewitcher) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:61]:
```suggestion
> When a workspace is converted to a Partner Workspace, Postman sends in-app and email notifications to all workspace members.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 25]
```diff
@@ -191,4 +191,4 @@ Workspace Admins and Partner Managers can delete Partner Workspaces.
 1. Select **Remove all partners**, then select **Update**.
 1. Select **Delete Workspace**.
 
-> Postman notifies all workspace members in-app and by email when you delete a Partner Workspace.
+> Postman notifies all workspace users in-app and by email when you delete a Partner Workspace.
```
Comment (petewitcher) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:194]:
```suggestion
> Postman notifies all workspace members in-app and by email when you delete a Partner Workspace.
```

File: src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md [hunk 26]
```diff
@@ -89,7 +89,7 @@ You can also watch specific elements within a team workspace to receive a notifi
 
 ## Next steps
 
-Now that you know how to collaborate in team workspaces, you might want more information about managing and working with workspaces:
+After learning how to collaborate in team workspaces, you might want more information about managing and working with workspaces:
```
Comment (petewitcher) [inline src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md:92]:
```suggestion
After learning how to collaborate in team workspaces, you can learn about managing and working with workspaces:
```

File: src/pages/docs/developer/collection-conversion.md [hunk 27]
```diff
@@ -49,6 +49,6 @@ Each of these Node.js modules is available through npm or by cloning directly fr
 
 ## Next steps
 
-Now that you understand the fundamentals of using converters to automate Postman Collection runs with your development, testing, or deployment pipeline, you might also want to use the Postman API in your CI/CD workflow:
+After learning the fundamentals of using converters to automate Postman Collection runs with your development, testing, or deployment pipeline, learn about using the Postman API in your CI/CD workflow:
```
Comment (petewitcher) [inline src/pages/docs/developer/collection-conversion.md:52]:
```suggestion
After learning the fundamentals of using converters to automate Postman Collection runs with your development, testing, or deployment pipeline, you can learn about using the Postman API in your CI/CD workflow:
```

File: src/pages/docs/getting-started/postman-account.md [hunk 28]
```diff
@@ -47,7 +47,7 @@ Before you sign up for a Postman account, either download and open the [desktop
 
 Postman prompts you to sign in or sign up. Select __Create Account__. You can sign up using an email address or using your Google account:
 
-* **Email** - Enter your email and create a username and password. Check the box to agree to the __Terms of Use__, which you can view by selecting the link, and select __Create free account__. Postman will send you an email to confirm your account. Follow the instructions in the email to complete your account setup.
+* **Email** - Enter your email and create a username and password. Check the checkbox to agree to the __Terms of Use__, which you can view by selecting the link, and select __Create free account__. Postman will send you an email to confirm your account. Follow the instructions in the email to complete your account setup.
```
Comment (petewitcher) [inline src/pages/docs/getting-started/postman-account.md:50]:
```suggestion
* **Email** - Enter your email and create a username and password. Select the checkbox to agree to the __Terms of Use__, which you can view by selecting the link, and select __Create free account__. Postman will send you an email to confirm your account. Follow the instructions in the email to complete your account setup.
```

**Review by HeatherStenson — APPROVED** [review]
I saw a few issues (noted in separate comments), but nothing breaking.

File: src/pages/docs/postman-flows/core-concepts/blocks-list.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/postman-flows/core-concepts/blocks-list.md:21]: Created https://postmanlabs.atlassian.net/browse/TW-809 for Pete to fix these later

Comment (mark-anthony-postman) [reply src/pages/docs/postman-flows/core-concepts/blocks-list.md:21]: Created a new Jira ticket to fix these later

File: src/pages/docs/postman-flows/core-concepts/blocks-list.md (hunk 6 above)
Comment (mark-anthony-postman) [reply src/pages/docs/postman-flows/core-concepts/blocks-list.md:115]: Created a new Jira ticket to fix these later

File: src/pages/docs/postman-flows/core-concepts/blocks-list.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/postman-flows/core-concepts/blocks-list.md:25]: Created a Jira ticket for Pete to fix later

File: src/pages/docs/administration/managing-your-team/managing-your-team.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md:92]: Created a new Jira ticket to correct this and all other suggestions made on this PR

File: src/pages/docs/getting-started/postman-account.md (hunk 28 above)
Comment (petewitcher) [reply src/pages/docs/getting-started/postman-account.md:50]: Resolved in TW-811

File: src/pages/docs/developer/collection-conversion.md (hunk 27 above)
Comment (petewitcher) [reply src/pages/docs/developer/collection-conversion.md:52]: Fixed in TW-811

File: src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md (hunk 26 above)
Comment (petewitcher) [reply src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md:92]: Resolved in TW-811

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md (hunk 25 above)
Comment (petewitcher) [reply src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:194]: Resolved in TW-811.

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md (hunk 24 above)
Comment (petewitcher) [reply src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:61]: Resolved in TW-811.

File: src/pages/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests.md (hunk 23 above)
Comment (petewitcher) [reply src/pages/docs/collaborating-in-postman/using-version-control/reviewing-pull-requests.md:126]: Resolved in TW-811.

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md (hunk 22 above)
Comment (petewitcher) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md:257]: Resolved in TW-811.

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md (hunk 21 above)
Comment (petewitcher) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md:58]: Resolved in TW-811.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md (hunk 20 above)
Comment (petewitcher) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:92]: Resolved in TW-811.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md (hunk 16 above)
Comment (petewitcher) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:91]: Resolved in TW-811.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md (hunk 15 above)
Comment (petewitcher) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:46]: Resolved in TW-811.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 29]
```diff
@@ -62,8 +62,8 @@ You can also select **Upload file(s)** to upload a new rule in valid YAML or JSO
...
+* To turn a governance rule off, select the toggle next to the rule name. You and team members won't see violations for this rule in your API's definition.
```
[the remark this answers is not in the document]
Comment (petewitcher) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:66]: Resolved in TW-811.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md (hunk 19 above)
Comment (petewitcher) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:65]: Resolved in TW-811.

File: src/pages/docs/administration/team-merge.md (hunk 12 above)
Comment (petewitcher) [reply src/pages/docs/administration/team-merge.md:164]: Resolved in TW-811.

File: src/pages/docs/administration/team-merge.md (hunk 8 above)
Comment (petewitcher) [reply src/pages/docs/administration/team-merge.md:109]: Resolved in TW-811.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md (hunk 3 above)
Comment (petewitcher) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md:92]: Resolved in TW-811.

**Pull request merged**